packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -433,22 +435,15 @@ describeWithDOM('mount', () => {
       expect(wrapper.find('.row + .row')).to.have.lengthOf(1);
     });
 
-    // React 15.2 warns when setting a non valid prop to an DOM element
-    describeIf(REACT013 || REACT014, 'unauthorized dom props', () => {
```

Comment:
This test was kind of a false positive. You could still query unauthorized DOM props by using the object syntax, and AFAICT it was only failing to find them because of how it was parsing the selector.

React <0.14 didn't just ignore syntactically invalid attributes, it ignored attributes that it didn't have whitelisted. To implement this behavior as it should be implemented we'd need to integrate that same whitelist.

Reply:
Can we add that whitelist in the react 0.14 and 0.13 adapters?

Reply:
Sure, OK to do in a follow up?

Reply:
Yes, totes ok to do in a followup